packages/node_modules/@webex/webex-core/src/lib/services/index.js

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
+ */
```

Comment:
I have a feeling that in the future, we will ask why this is a plugin within core instead of a separate one. 

Could you add a note explaining why we implemented it this way and what the limitations of the "normal" plugin route was?